libs/sdk-backend-spi/src/workspace/attributes/elements/index.ts

```diff
@@ -20,13 +69,16 @@ export interface IElementsQueryOptions {
     filter?: string;
 
     /**
+     * @privateRemarks
      * TODO what is this doing?
+     * @deprecated do not use.
      */
     prompt?: string;
 
     /**
-     *  With this option you can specify concrete attribute elements uris to load.
-     *  This is commonly used to preload selected elements in the attribute filter
+     * With this option you can specify concrete attribute elements uris to load.
+     * This is commonly used to preload selected elements in the attribute filter.
```

Comment:
The second line of the comments should be in the remarks tag, it says some detailed info about the property